modules/site-health/admin-site-health.php

```diff
@@ -0,0 +1,248 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Class PLL_Admin_Site_Health to add debug info in WP Site Health
+ *
+ * @since 2.8
+ * @link https://make.wordpress.org/core/2019/04/25/site-health-check-in-5-2/
+ */
+class PLL_Admin_Site_Health {
+
+	public $model;
+
+	/**
+	 * PLL_Admin_Site_Health constructor.
+	 *
+	 * @since 2.8
+	 */
+	public function __construct( &$polylang ) {
+		// Information tab
+		add_filter( 'debug_information', array( $this, 'info_options' ), 15 );
+		add_filter( 'debug_information', array( $this, 'info_languages' ), 16 );
+
+		// tests Tab
+		add_filter( 'site_status_tests', array( $this, 'is_homepage' ) );
+
+		$this->model = &$polylang;
+	}
+
+	/**
+	 * Return a list of key to exclude from site health informations
+	 *
+	 * @return array list of option key to ignore
+	 * @since   2.8
+	 */
+	public function exclude_key() {
+		return array(
+			'flag',
+			'host',
+			'taxonomy',
+			'description',
+			'parent',
+			'filter',
+			'uninstall',
+			'first_activation',
+			'custom_flag',
+		);
+	}
+
+	/**
+	 * Add Polylang Options to Site Health Informations tab.
+	 *
+	 * @param array $debug_info array options to display.
+	 * @return array
+	 * @since   2.8
+	 */
+	public function info_options( $debug_info ) {
+		$options = get_option( 'polylang' );
+		$fields = array();
+		foreach ( $options as $key => $value ) {
+			if ( in_array(
+				$key,
+				$this->exclude_key()
+			)
+				) {
+				continue;
+			}
+			if ( ! is_array( $value ) ) {
+				if ( false === $value ) {
+					$value = '0';
+				}
+				$fields[ $key ]['label']   = $key;
+				$fields[ $key ]['value']   = $value;
+			} else {
+				if ( empty( $value ) ) {
+					$fields[ $key ]['label']   = $key;
+					$fields[ $key ]['value']   = '0';
+				} else {
+					switch ( $key ) {
+						case 'post_types':
+							$fields[ $key ]['label'] = $key;
+							$fields[ $key ]['value'] = implode( ', ', $this->model->model->get_translated_post_types() );
+							break;
+						case 'taxonomies':
+							$fields[ $key ]['label'] = $key;
+							$fields[ $key ]['value'] = implode( ', ',
+								$this->model->model->get_translated_taxonomies() );
+							break;
+						case 'nav_menus':
+							$current_theme = wp_get_theme();
+							foreach ( $value as $menus ) {
+								if ( key( $value ) !== $current_theme->stylesheet ) {
+									continue;
+								}
```

Comment:
I believe there's only one entry in the `$value` array that will satisfy this condition, and we already know its key. If so, it would be easier to read if you simply access the entry with `$value[ $current_theme->stylesheet ]`, instead of looping to skip all the cases.